### PR TITLE
CD workflow connect to vault

### PR DESCRIPTION
**Pull request by kvs96**

### CODE REVIEW

File: .github/workflows/CD.yml [hunk 1]
```diff
@@ -1,78 +1,84 @@
...
+            kv/telegram_bots/alerts chat_id | TG_CHAT ;
+            kv/telegram_bots/alerts token | TG_TOKEN
 
-      - name: Install Info
-        run: |
-            ls -l .
-            ls -l ../..
-            pwd
+      - name: Checkout
+        uses: actions/checkout@v3.0.2
```
Comment (shamilsan) [inline .github/workflows/CD.yml:54]: I don't see the reason to choose a concrete tag. Better to use a branch with the latest release within the major version. See for example: https://github.com/actions/checkout/
```suggestion
        uses: actions/checkout@v3
```

File: .github/workflows/CD.yml [hunk 2]
```diff
@@ -1,78 +1,84 @@
...
 
   deploy:
     runs-on: ubuntu-latest
-    needs: docker
+    needs: build
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+      - name: Read secrets
+        uses: hashicorp/vault-action@v2.4.1
```
Comment (shamilsan) [inline .github/workflows/CD.yml:34]: Is this vault backed up and available for somebody except you? It should be protected against the bus factor.

**Comment by breathx** [comment]
Is it stale PR? @kvs96

**Comment by shamilsan** [comment]
Reopen it if needed.

**Pull request closed without merging**